packages/js/src/ui/context/InboxContext.tsx

```diff
@@ -42,6 +44,7 @@ export const InboxProvider = (props: InboxProviderProps) => {
     ...STATUS_TO_FILTER[NotificationStatus.UNREAD_READ],
     tags: props.tabs.length > 0 ? props.tabs[0].value : [],
   });
+  const [hideBranding, setHideBranding] = createSignal(false);
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** ‏Added it to the inboxContext, let me know if i should move it to a different context or a new one

Reply:
I think this is the right place for now.